src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Button from './Button';
+
+export default function App() {
+  return (
+    <div id="buttonLayout">
+      <Button />
+      <ButtonPanel />
+    </div>
+  );
```

Comment:
- As stated here 
![image](https://user-images.githubusercontent.com/46115725/94041976-924f2600-fd90-11ea-94cc-685bc5d5e9b1.png)
We should be importing button panel and display. Please do make the adjustments.